tests/benchdnn/graph/graph_memory.cpp

```diff
@@ -143,7 +148,11 @@ dnnl::graph::tensor dnn_graph_mem_t::make_graph_tensor(
     dnnl_memory_get_data_handle(mem_.m_, &data_handle);
     dnnl::graph::logical_tensor graph_lt(lt.id_, lt.get_data_type(), lt.shape_,
             str2layout(lt.layout_type_), lt.get_property_type());
-    dnnl::graph::tensor ret(graph_lt, get_graph_engine(), data_handle);
+    bool is_host_scalar = lt.property_type_ == "host_scalar";
```

Comment:
I'd introduce a method in `deserialized_lt_t` for this line and use a function everywhere.

Reply:
incorporated all the comments, thanks!